src/lib/data/analytics.js

```diff
@@ -0,0 +1,116 @@
+import Analytics from 'analytics-node';
+import { isStyledComponent } from "styled-components";
+import { nanoid } from 'nanoid';
+import { debounce, get, split, truncate } from 'lodash'
+
+let segment
+let anonymousUserId
+let userIdCreatedAt
```

Comment:
missing semicolons

Reply:
they are optional no?

Reply:
for consistency we always use them, and most of the time it's handled automatically since at Pagoda we use `prettier`. If you leave them off in your PR, they will show up as a diff for others when auto-formatting runs the next time they touch this file